completions/gem.completion.sh

```diff
@@ -34,7 +34,7 @@ _gem() {
       return 0
       ;;
   esac
-  local commands=(build cert check cleanup contents dependency environment fetch generate_index help install list lock outdated owner pristine push query rdoc search server sources specification stale uninstall unpack update which)
+  local commands=(build cert check cleanup command contents dependency environment fetch generate_index help install list lock outdated owner pristine push query rdoc search server sources specification stale uninstall unpack update)
```

Comment:
What is this change? According to [this page](https://guides.rubygems.org/command-reference/), `gem command` doesn't exist while `gem which` does exist.